tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/application/openshift/RhSsoTemplateOpenShiftApplication.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.intersmash.tools.application.openshift;
+
+import java.nio.file.Path;
+
+import org.jboss.intersmash.tools.application.openshift.template.RhSsoTemplate;
+
+/**
+ * End user Application interface which presents RH-SSO template application on OpenShift Container Platform.
+ * <p>
+ * RH-SSO application that is supposed to run on OpenShift needs to implement this interface.
+ * Usage:
+ * <pre>
+ *     &#064;Appsint(
+ *     		&#064;Service(RhssoApp.class)
+ *     })
+ * </pre>
+ * The application will be deployed by:
+ * <ul>
+ *     <li>{@link RhSsoTemplateOpenShiftProvisioner}</li>
+ * </ul>
+ * <p>
+ * See {@link RhSsoTemplate} for available templates the
+ * application can represent.
+ */
+public interface RhSsoTemplateOpenShiftApplication extends TemplateApplication<RhSsoTemplate>, HasSecrets {
+
+	default String getName() {
+		return "rh-sso";
+	}
+
+	/**
+	 * Realm configuration in json format for Keycloak partial import
+	 * <p>
+	 * https://www.keycloak.org/docs/9.0/server_admin/index.html#importing-a-realm-from-exported-json-file
+	 * <p>
+	 * Requires template parameters SSO_REALM, SSO_SERVICE_USERNAME, SSO_SERVICE_PASSWORD to be set
+	 *
+	 * @return Instance of {@link Path} representing a YAML definition for the desired realm configuration
+	 */
+	default Path getRealmConfigurationFilePath() {
+		return null;
```

Comment:
@fabiobrz importing a realm isn't needed, right?

Reply:
It depends, there are internal tests relying on this specific provisioner, hence I am going to check and get beck here, thanks for now!

Reply:
Confirmed, it is used by existing EAP7 + RHSSO tests.